api/build.gradle

```diff
@@ -58,18 +68,19 @@ dependencies {
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-web'
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa'
     compile group: 'org.springframework.boot', name: 'spring-boot-starter-security'
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
+    compile (group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'){
+        exclude(module:"micrometer-core")
```

Comment:
We don't need exclusion as we have explicitly added a resolution strategy

Reply:
have removed the exclusion